account_invoice_merge/models/account_move.py

```diff
@@ -154,9 +154,11 @@ def make_key(br, fields):
 
                 if o_line:
                     # merge the line with an existing line
+                    o_line["name"] += "\n" + invoice_line.name
```

Comment:
or better

```suggestion
                    o_line["name"] += ("\n" + invoice_line.name) if invoice_line.name not in o_line["name"] else ''
```

to handle cases where the names are the same?